index.bs

```diff
@@ -2453,6 +2453,8 @@ Unanimous Short Circuit</h5>
 	the Team recommends a resolution
 	and every potential member of a Council who is not renouncing their seat
 	votes affirmatively (no abstentions) to adopt this resolution.
+	As an exception, Tim Berners-Lee <em class=rfc2119>may</em> abstain
```

Comment:
```suggestion
	As an exception, the voting affirmatively requirement does not apply to Tim Berners-Lee, 
	who <em class=rfc2119>may</em> omit his vote or abstain.
```

Reply:
This edit no longer makes sense, since the underlying text has been changed by #885.